workflow/pred_temperature_ridge.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<p>When I look at this without running, it's a bit odd that the start_year is the last index. Perhaps:</p><p> </p><pre class="ql-syntax" spellcheck="false"># get 70% of instance as training
years = sorted(calendar.get_intervals().index)
train_samples = round(len(years) * 0.7)
start_year = years[0]
</pre><p> </p><p>*Change made in commit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>